python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -359,9 +395,17 @@ def _load_data(
             an external string via this parameter which will be added to the
             seed.
 
-        :return:
+        :returns: True if cached data was loaded, False if not.
         """
 
+        if not self._hierarchy_is_supported:
+            self.clear()
+            root = self.invisibleRootItem()
+            item = QtGui.QStandardItem("WARNING: SG version must be v7.0.2 or higher to use the Hierarchy Model")
```

Comment:
My thought was that as long as `WARNING` showed up, it would be sufficient and most views would have scrolling/cell expansion to show the rest. I'm definitely open to change it though.